docs/application/native/guides/multimedia/radio.md

```diff
@@ -80,13 +80,12 @@ To enable your application to use the radio functionality:
        mRadioInfo* mRadio = (mRadioInfo *)userdata;
 
        switch (code) {
-       case RADIO_INTERRUPTED_COMPLETED:
-           /* Application, which was the source of the conflict, is now closed */
-           /* Restart the radio playback here */
+       case RADIO_INTERRUPTED_BY_RESOURCE_CONFLICT:
+           /* Radio listening is interrupted by resource */
+           /* Release the application resources or save the current state here */
            break;
        default:
-           /* Radio listening is interrupted */
-           /* Release the application resources or save the current state here */
+           /* Other type isn't handled (Other types have been removed since tize_5.0)*/
```

Comment:
Question: What is other type? Is it type or types?
Change to (verify the meaning):

Other types are not supported from Tizen 5.0 onwards.